styles/modifier.css

```diff
@@ -0,0 +1,50 @@
+.rss001 {
```

Comment:
Try to give a more convenient name for the class, as a user it's confusing for me to understand it.